### DISCUSSION TITLE
SFTP Connection on MAC M2 not working

### DISCUSSION THREAD

**Discussion by Locoza** [iterate-ch/cyberduck#14609] [Q&A]
Hi, I was wondering if anyone could help I have an SFTP server connection issue on my new Macbook, the login works on my old Intel Mac and on my friend m1 air but on my M2 the connection just does nothing I don't get an error or anything it just loads for a split second and then stays the same without making the connection to the server. 

Any advice would be appreciated. 

Thnaks

**Comment by tatze96** [comment]
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), re-create the described behavior, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14609 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
